### ISSUE TITLE
share on Google Drive

### ISSUE THREAD

**Issue by TomKomar** [BJTUJia/person_reID_DualNorm#2]
Hi, Could you please upload the items you shared on Baidu onto Google Drive? There's no language switch on Baidu and navigating it blindly always takes me to Baidu Drive installer (.exe) which also has no language switch.

**Comment by astroseger** [comment]
On ubuntu it seems impossible to download from Baidu (in Russia). I even failed to download the client which apparently you need to download something from Baidu... 

@TomKomar Have you manage do download pre-trained models? Could you share them with me?

**Comment by TomKomar** [comment]
does you gmail begin with your github nick?

**Comment by astroseger** [comment]
answer is positive.

**Comment by BJTUJia** [comment]
Sorry for the late response. I was busy with my other project and haven't checked Github in a while. Anyway, here are the Google Drive links for the shared models: trainedmodels: https://drive.google.com/open?id=1Gy96vKH60ML9fk2znnZmz5LghS08-M0R Test set:https://drive.google.com/open?id=1-5JqXxqQ14MCngGLMPqiA-zbI2CUBFxV. Enjoy!
Many thanks for your interest in our work!

**Comment by ppriyank** [comment]
VIPeR doesn't seem to exist on google drive anymore.  Can you please re-share it? The Baidu link doesn't seem to translate, hence I can't download it from there.
